src/constants.js

```diff
@@ -224,3 +224,7 @@ export const PlatformTypes = {
   TWITCH_CLIPS: 2,
   YOUTUBE: 3,
 };
+
+export const ITEM_HEIGHT = 48;
+
+export const EmoteCategoriesOrderStorageKey = 'emote-categories-order';
```

Comment:
use `PascalCase` when defining constant enums or objects, as well as classes
use `UPPER_CASE` when defining other constants
use `camelCase` when defining variables and functions